pyth_observer/calendar.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+EQUITY_OPEN = datetime.time(9, 30, 0)
+EQUITY_CLOSE = datetime.time(16, 0, 0)
+EQUITY_EARLY_CLOSE = datetime.time(13, 0, 0)
+EQUITY_HOLIDAYS = ["17-01-2022", "08-02-2022", "21-02-2022", "15-04-2022", "30-05-2022",
+                   "20-06-2022", "04-07-2022", "05-09-2022", "24-11-2022", "25-11-2022", "26-12-2022"]
+EQUITY_EARLY_HOLIDAYS = ["08-02-2022", "25-11-2022"]
+
+
+class Calendar():
```

Comment:
This just seems like a very odd approach. You're converting datetimes to strings to compare them. If feasible, please try to use native datetimes. This just screams unpythonic to me as-is.

Reply:
fixed!

Reply:
(nitpick) Maybe rename this to like `HolidayCalendar` ?  This name is a bit too generic.